### ISSUE TITLE
setLocale() facing TypeError: The header content contains invalid characters

### ISSUE THREAD

**Issue by ShinyLeee** [vazco/meteor-universe-i18n#39]
I want to get Chinese translation,I got the corresponding translation after setLocale() , but this happened in my console.
![image](http://o97tuh0p2.bkt.clouddn.com/61C.tmp.jpg)

![image](http://o97tuh0p2.bkt.clouddn.com/B9FD.tmp.jpg)

Does it mean my translation files not named correctly?

My translation files are located in **i18n/en.i18n.yml, i18n/zh-CN.i18n.yml**

in the server

```
TypeError: The header content contains invalid characters
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:351:13)
    at ServerResponse.res.setHeader (C:\Users\Administrator\AppData\Local\.meteor\packages\webapp\1.3.11\npm\node_modules\connect\lib\patch.js:134:22)
    at ServerResponse.setWriteHeadHeaders (C:\Users\Administrator\AppData\Local\.meteor\packages\webapp\1.3.11\npm\node_modules\on-headers\index.js:82:19)
    at ServerResponse.writeHead (C:\Users\Administrator\AppData\Local\.meteor\packages\webapp\1.3.11\npm\node_modules\on-headers\index.js:41:36)
    at Object.handle (meteor://�app/packages/universe:i18n/server/handler.js:42:17)
    at next (C:\Users\Administrator\AppData\Local\.meteor\packages\webapp\1.3.11\npm\node_modules\connect\lib\proto.js:174:15)
    at Function.app.handle (C:\Users\Administrator\AppData\Local\.meteor\packages\webapp\1.3.11\npm\node_modules\connect\lib\proto.js:182:3)
    at Object.fn [as handle] (C:\Users\Administrator\AppData\Local\.meteor\packages\webapp\1.3.11\npm\node_modules\connect\lib\proto.js:79:14)
    at next (C:\Users\Administrator\AppData\Local\.meteor\packages\webapp\1.3.11\npm\node_modules\connect\lib\proto.js:174:15)
    at Object.WebAppInternals.staticFilesMiddleware (meteor://�app/packages/webapp/webapp_server.js:369:5)
```

Here are my codes.

main.js

```
import { Meteor } from 'meteor/meteor';
import { render } from 'react-dom';
import i18n from 'meteor/universe:i18n';
import utils from '../imports/api/utils.js';
import renderRoutes from './routes.jsx';

if (Meteor.isClient) {
  Meteor.startup(() => {
    i18n.setLocale(utils.language());
    render(renderRoutes(), document.getElementById('app'));
  });
}
```

Forgive my poor English.

**Comment by shihanjun19** [comment]
+1

**Comment by cristo-rabani** [comment]
Hello,

Could you tell me what is your correct url? 
Check if `location.origin + '/' === Meteor.absoluteUrl()` gives true.
If result is `false` you should set up meteor url

best

**Comment by ShinyLeee** [comment]
@cristo-rabani 
Thanks for replying.

the result is true..

I can get translation which I write in i18n/zh-CN.i18n.yml,but console always throw error.

**Comment by moranda** [comment]
i encounter the same problem, any solution found?

**Comment by wansj** [comment]
If you had imported the en.i18n.json and zh.i18n.json files in startup/client/index.js, this problem can be resolved by setLocale('zh-CN',{noDownload:false} ).

**Comment by cristo-rabani** [comment]
This problem seems to be connected with request encoding

**Comment by damien-monni** [comment]
Same problem for me.
I have 2 translation files **en** and **fr**.
If I don't use setLocal the application is by default in english and everything works well.
When I use `i18n.setLocale('fr');` I got the same problem.

I am running Windows 10.

I tried a Meteor fresh install and noticed that when I uninstall meteor, universe_i18n folder in C:\Users\myusername\AppData\Local\.meteor\packages isn't remove. If I try to manually remove it I can't because filenames are too long... thanks Windows.
Coud it be related to the problem?
